localgov_events.info.yml

```diff
@@ -1,6 +1,6 @@
 name: LocalGov Events
 description: Provides events for LocalGov Drupal.
-core_version_requirement: ^8.9 || ^9
+core_version_requirement: ^9 || ^10
```

Comment:
This removes 9 from the matrix, probably because of the dependency on Geo, so probably shouldn't be claiming 9 compat here? Putting as a D10 only release in the maxtrix.